internal/venice-common/src/main/java/com/linkedin/venice/stats/dimensions/VeniceMetricsDimensions.java

```diff
@@ -0,0 +1,48 @@
+package com.linkedin.venice.stats.dimensions;
+
+import static com.linkedin.venice.stats.VeniceOpenTelemetryMetricFormat.CAMEL_CASE;
+import static com.linkedin.venice.stats.VeniceOpenTelemetryMetricFormat.PASCAL_CASE;
+import static com.linkedin.venice.stats.VeniceOpenTelemetryMetricFormat.SNAKE_CASE;
+import static com.linkedin.venice.stats.VeniceOpenTelemetryMetricsRepository.transformMetricName;
+import static com.linkedin.venice.stats.VeniceOpenTelemetryMetricsRepository.validateMetricName;
+
+import com.linkedin.venice.stats.VeniceOpenTelemetryMetricFormat;
+
+
+public enum VeniceMetricsDimensions {
```

Comment:
Can we remove `venice.` prefix from these? These should be getting reflected in the payload, and is redundant since the metric names will have the `venice.` prefix

Reply:
There can be other dimensions added by layers in between Venice OTel and the actual receivers (for example: if fluent bit agent is used as the exporter in hosts running Venice). Adding `venice` explicitly here 
1. says Venice added these dimensions and not other players down the line
2. makes it differentiate similar dimensions (for instance: `venice.response.status_code_category` and `http.response.status_code_category`)
The above 2 reasons made me add `venice` prefix for the dimensions. WDYT?

Reply:
As discussed, keeping the namespace in both metrics and dimensions for now. Will check further on this.